test/helpers/getCompiler.js

```diff
@@ -6,9 +6,11 @@ import { createFsFromVolume, Volume } from 'memfs';
 import MiniCssExtractPlugin from '../../src';
 
 export default (fixture, loaderOptions = {}, config = {}) => {
+  const { outputFileSystem, ...cnfg } = config;
```

Comment:
Not related to PR fix, I just wanted to look on output when was testing locally and found a bug with `outputFileSystem` option..